src/config/session-management.md

```diff
@@ -0,0 +1,32 @@
+# Session and Seat Management
+
+Session and seat management is not necessary for every setup, but it can be used
+to safely create temporary runtime directories, provide access to hardware
+devices, and control system shutdown.
+
+## D-Bus
+
+D-Bus is an IPC (inter-process communication) mechanism used by userspace
+software in Linux. There are two uses for it: as a system bus, and as a user
+bus. To use it as a system bus, you should [enable](./services/index.md) the
+`dbus` service. To use it as a user bus, you can start a given program (usually
+a window manager or interactive shell) with
+[dbus-run-session(1)](https://man.voidlinux.org/dbus-run-session.1). Most
+desktop environments, if launched through an adequate display manager, will
+launch a D-Bus session themselves.
+
+## elogind
+
+[elogind(8)](https://man.voidlinux.org/elogind.8) is a standalone version of
+`systemd-logind`, a service to manage user logins. This service provides
+necessary features for most desktop environments and Wayland compositors. To
+make use of its features, install the `elogind` package and enable the `dbus`
+service. This might require a reboot.
+
+### Troubleshooting
+
+If you're having any issues with `elogind`, [enable](./services/index.md) its
+service, as waiting for a D-Bus activation can lead to issues.
+
+If you have the [acpid(8)](https://man.voidlinux.org/acpid) service enabled,
```

Comment:
Or you can `ignore` the acpid functions in elogind in `/etc/elogind/logind.conf`

Reply:
Do you have some idea on how to explain this? I couldn't find an explicit list of what acpid functions were available in `logind.conf(5)`.

Reply:
I added mostly what you said, if you have some function list we could consider adding it.

Reply:
basically everything that can accept "ignore" should all say "ignore". Like the Handle* section, for example.

Reply:
What do you think of my phrasing now?

Reply:
options related to acpi? acpi is the specification, while acpid is the daemon

Reply:
How about
```
By default, `elogind` listens for and processes ACPI events related to lid-switch activation and presses of power, suspend and hibernate keys. This will conflict with `acpid` if it is installed an enabled. Either disable `acpid` when enabling `elogind` or modify `/etc/elogind/logind.conf` to ignore these events within `elogind`. The `HandlePowerKey`, `HandleSuspendKey`, `HandleHibernateKey` and multiple `HandleLidSwitch` options in [logind.conf(5)](https://man.voidlinux.org/logind.conf.5) should be set to `ignore` to stop `elogind` from responding to ACPI events.
```

Reply:
@travankor "options related to controlling acpi events", possibly.

@ahesford do you think those will remain valid? I don't want to let the docs grow stale.

Reply:
If you want to be agnostic, then
```
Either disable `acpid` when enabling `elogind` or configure `elogind` to `ignore` ACPI events
in `/etc/elogind/logind.conf`. The [logind.conf(5) man page](https://man.voidlinux.org/logind.conf.5)
describes several configuration options, all starting with `Handle`, that should be set to
`ignore` to avoid interfering with `acpid`.
```

Reply:
Awesome! Thank you both :D

Reply:
Mixed it up a bit, I think it looks good now.